rodan/jobs/helloworld/helloworld.py

```diff
@@ -29,6 +29,47 @@ def run_my_task(self, inputs, settings, outputs):
         outfile.close()
         return True
 
+class HelloWorld_1PortType_MultiPorts(RodanTask):
+    name = 'Hello World MultiTasks'
+    author = 'Silver'
+    description = "A demo of showing the multiports job functions, didn't add list type parsing"
+    settings = {}
+    enabled = True
+    category = "Test"
+    interactive = False
+
+    input_port_types = (
+        {
+            'name': 'Text input', 
+            'minimum': 0, 
+            'maximum': 10, 
+            'resource_types': ['text/plain']
+        },
+    )
+    output_port_types = (
+        {
+            'name': 'Text output', 
+            'minimum': 1, 
+            'maximum': 10, 
+            'resource_types': ['text/plain']
+        },
+    )
+
+    def run_my_task(self, inputs, settings, outputs):
+        for output_type in outputs:
+            for output in outputs[output_type]:
+                outfile = open(output["resource_path"], "w")
+                self.WriteInputFile(inputs, outfile)
```

Comment:
🚫 Lower case `w`.

🚫  In Python, we usually use snake_case as variable/function names and use CamelCase for classes. Please rename it as `write_input_file`.